include/roboptim/core/util.hxx

```diff
@@ -170,7 +170,7 @@ namespace roboptim
     ROBOPTIM_ASSERT (startRow + b.rows () <= m.rows ());
     ROBOPTIM_ASSERT (startCol + b.cols () <= m.cols ());
 # if __cplusplus < 201103L || !defined (__GXX_EXPERIMENTAL_CXX0X__)
-    ROBOPTIM_ASSERT (int (M::IsRowMajor) == int (B::IsRowMajor));
+    ROBOPTIM_ASSERT ( ROBOPTIM_UNUSED int (M::IsRowMajor) == int (B::IsRowMajor));
```

Comment:
I checked that with Visual Studio, this is the only position working for ROBOPTIM_UNUSED.
So I guess that, we need to add a #ifdef WIN32...